main.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import FastAPI
+
+from city_temperature_app.api import city, temperature

```

Comment:
[Improvement] You can organize your imports more cleanly by using aliasing, like this:

```
from city_temperature_app.api.city import router as city_router
```

This approach makes your code more readable and easier to manage, especially when you have multiple routers or modules. Additionally, by consistently applying such patterns, your project will maintain a clear and professional structure, enhancing both collaboration and maintainability.